README.md

```diff
@@ -94,72 +74,57 @@ export MODIN_ENGINE=dask  # Modin will use Dask
 This can also be done within a notebook/interpreter before you import Modin:
 
 ```python
-import os
-
-os.environ["MODIN_ENGINE"] = "ray"  # Modin will use Ray
-os.environ["MODIN_ENGINE"] = "dask"  # Modin will use Dask
-
 import modin.pandas as pd
+from modin.config import Engine
+
+Engine.put("ray")  # Modin will use Ray
+Engine.put("dask")  # Modin will use Dask 
 ```
 
-Check [this Modin docs section](https://modin.readthedocs.io/en/latest/UsingOmnisci/index.html) for Omnisci engine setup.
+Check [this Modin docs section](https://modin.readthedocs.io/en/latest/development/using_omnisci.html) for Omnisci engine setup.
 
-**Note: You should not change the engine after you have imported Modin as it will result in undefined behavior**
+**Note: You should not change the engine after your first operation with Modin as it will result in undefined behavior.**
 
-##### Which engine should I use?
+#### Which engine should I use?
```

Comment:
nit: the bolded `Note` above is almost the same font as this subsection, so it's a bit hard to see.